### PR TITLE
Grab open loans from subgraphs

### PR DESCRIPTION
**Pull request by bachstatter**

### CODE REVIEW

File: containers/Loans/Loans.tsx [hunk 1]
```diff
@@ -71,6 +72,21 @@ function Container() {
...
+	);
+
+	const subgraphOpenLoansKey = subgraphOpenLoansQuery.data
+		? JSON.stringify(
+				subgraphOpenLoansQuery.data.map((x) => {
+					const id = x.id.replace(/-\w+/, ''); //remove -sETH from id
+					return { ...x, id };
+				})
+		  )
+		: '';
```
Comment (Rickk137) [inline containers/Loans/Loans.tsx:88]: I don't understand the reason behind stringifing the array and then parsing it down in the code

File: containers/Loans/Loans.tsx [hunk 2]
```diff
@@ -91,11 +107,13 @@ function Container() {
...
-				loanContracts,
-				loanStateContracts,
+				erc20LoanContract,
+				ethLoanContract,
+				erc20LoanStateContract,
+				ethLoanStateContract,
 				address,
-				network,
 				isL2,
+				subgraphOpenLoans: subgraphOpenLoansKey ? JSON.parse(subgraphOpenLoansKey) : [],
```
Comment (Rickk137) [inline containers/Loans/Loans.tsx:116]: here we are parsing the subgraphOpenLoansKey. we could simply store it as array instead of stringify

Comment (Rickk137) [inline containers/Loans/Loans.tsx:116]: here we are parsing the subgraphOpenLoansKey. we could simply store it as array instead of stringify

**Review by Rickk137 — CHANGES_REQUESTED** [review]

File: containers/Loans/Loans.tsx (hunk 1 above)
Comment (bachstatter) [reply containers/Loans/Loans.tsx:88]: We want to rerun this useEffect when there is a change in openLoans. You cant put and array in the `useEffect`s dependency array

File: containers/Loans/Loans.tsx (hunk 2 above)
Comment (bachstatter) [reply containers/Loans/Loans.tsx:116]: See comment above

File: containers/Loans/Loans.tsx (hunk 1 above)
Comment (Rickk137) [reply containers/Loans/Loans.tsx:88]: Nice, didn't notice that

**Review by Rickk137 — APPROVED** [review]

**Pull request merged**